docker-app/qfieldcloud/core/serializers.py

```diff
@@ -154,7 +153,6 @@ class Meta:
             "data_last_packaged_at",
             "data_last_updated_at",
             "can_repackage",
-            "needs_repackaging",
```

Comment:
You need to ensure that `needs_repackaging` is delivered in the API, otherwise you will break QField.